src/signals/incident/components/form/MapSelectors/Asset/Selector_v2_removeafterfinishepic5440/DetailPanel/styled.tsx

```diff
@@ -27,9 +26,29 @@ export const StyledAssetList = styled(AssetList)`
   }
 `
 
-export const StyledButton = styled(Button)`
-  margin-top: ${themeSpacing(4)};
+export const StyledButton = styled(Button)<{
+  $isMobile?: boolean
+  $hasSubmitButton?: boolean
+}>`
+  position: fixed;
```

Comment:
better yet to use position sticky. now its positioned to the screen, with sticky ''offset relative to its nearest scrolling ancestor and [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block)''